cerebrium/environments/config-files.mdx

```diff
@@ -0,0 +1,155 @@
+---
+title: Using Config Files
+description: Using config files to configure Cortex deployments easily and quickly
+---
+
+After you've created your Cortex project, you may find that you need more control over your deployment than the default `cerebrium deploy` command provides.
+For example, you may want to specify the number of GPUs to use, the amount of memory to use or even the version of Python for your environment.
+These settings and more are all configurable using config files.
+
+## Deployment Parameters
+
+Deployment parameters are govern the persistent environment in which your model is deployed.
```

Comment:
"Deployment parameters are govern the persistent environment" I think remove the "are"

Reply:
Whhoooops. Thanks Jono!